src/message/__tests__/messageReducers-test.js

```diff
@@ -0,0 +1,300 @@
+import deepFreeze from 'deep-freeze';
```

Comment:
Does this include both the changes and the rename?
Doing it in two steps will make the diff readable.

Reply:
Hmm, yeah, I thought about doing that, but then didn't because there are already a lot of commits that really shouldn't be separate if it weren't for readability, and someone who want to see the diff can still easily generate it. But I guess I can just as well do it.

Reply:
> there are already a lot of commits that really shouldn't be separate if it weren't for readability

True. I'd suggest squashing together the commits that are all doing the same test fixes on different test files. The result is a bit big (8 files changed, 291 insertions(+), 205 deletions(-)), but it's highly uniform and boring -- and each file and indeed each test can be read and understood independently -- so that's OK.

As always, I don't really at all mind having commits split "too finely" -- squashing them back together is just about the easiest kind of change for me to make, and if I were otherwise about to merge this I'd simply do that. Far better to err on the side of splitting too finely than too coarsely. :slightly_smiling_face: 

(Maybe leave the `flagsReducer` one separate, to keep the handy extra info in its commit message right next to the change? But I think it'd also be fine to squash it and just include that paragraph in the squashed commit message.)